__tests__/app.test.js

```diff
@@ -22,4 +22,37 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics }  = body;
+        expect(Array.isArray(topics)).toBe(true);
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty('slug');
+          expect(topic).toHaveProperty('description');
+        })
+      });
+    });
+  test("404: Responds with Endpoint Not Found when the endpoint is incorrect", () => {
```

Comment:
Nice test, please move it out of the topics describe block as this test is not related to any particular endpoint.